### ISSUE TITLE
weird destructuring in onPostBuild 

### ISSUE THREAD

**Issue by logemann** [algolia/gatsby-plugin-algolia#115]
the code:

```
exports.onPostBuild = async function ({ graphql }, config) {
  const {
    appId,
    apiKey,
    queries,
    concurrentQueries = true,
    skipIndexing = false
  } = config;
```

doesnt work on my gatsby installation because the parameters you are looking for are in "config.options" and not "config". But this is such a hughe bug, i somehow wonder if its just me :-)

**Comment by Haroenv** [comment]
Thanks for posting this issue, which version of Gatsby do you use? Have you tried the version in the example in this repo, does it make a difference? If you could make a reproducible example I'd love to take a look to what's going on.

**Comment by logemann** [comment]
Gatsby CLI version: 2.16.1
Gatsby version: 2.29.0

I am using the very same config as in the example i think. And i think the index builder code doesnt use the "indexName" config option from gatsby-config.js. When leaving out the indexName in the queries array, the default indexName from config should be used....

**Comment by Haroenv** [comment]
I think this should be fixed by #127
